src/ensembl/src/shared/components/upload/Upload.tsx

```diff
@@ -176,6 +177,7 @@ const Upload = (props: UploadProps) => {
 };
 
 Upload.defaultProps = {
+  id: 'upload_' + generateId(),
```

Comment:
This is much better, but why have the id at all? What happens if we remove it?

Reply:
Are you saying we can set the id's only if it is passed in?

Reply:
I am not sure we even need to set an id on the input field at all. Do we?

Reply:
I've removed it 🤞